common.go

```diff
@@ -18,89 +18,83 @@ package ledger_cosmos_go
 
 import (
 	"encoding/binary"
+	"errors"
 	"fmt"
+	"strconv"
+	"strings"
 )
 
-// VersionInfo contains app version information
-type VersionInfo struct {
-	AppMode uint8
-	Major   uint8
-	Minor   uint8
-	Patch   uint8
+type VersionResponse struct {
+	AppMode   uint8 // 0: release | 0xFF: debug
+	Major     uint8
+	Minor     uint8
+	Patch     uint8
+	AppLocked uint8
+	TargetId  uint32
 }
 
-func (c VersionInfo) String() string {
-	return fmt.Sprintf("%d.%d.%d", c.Major, c.Minor, c.Patch)
-}
-
-// VersionRequiredError the command is not supported by this app
-type VersionRequiredError struct {
-	Found    VersionInfo
-	Required VersionInfo
+type AddressResponse struct {
+	pubkey  []byte // 0: release | 0xFF: debug
+	address string
 }
 
-func (e VersionRequiredError) Error() string {
-	return fmt.Sprintf("App Version required %s - Version found: %s", e.Required, e.Found)
+type SignatureResponse struct {
+	signatureDER []byte
```

Comment:
Maybe here you need a getter for `signatureDER` if you don't want to make that field public

Reply:
There isn't any problem if this field is public or not. At the moment, the app only delivers the signature of the transaction in DER format. It's just a clarification if the user needs to verify the signature.